verifier/src/report_body.rs

```diff
@@ -2,8 +2,11 @@
 
 //! Verifiers which operate on the [`ReportBody`]
 
-use crate::{VerificationResult, VerificationResultMetadata, Verifier};
-use core::fmt::Debug;
+use crate::{
+    ResultMessage, VerificationResult, VerificationResultMetadata, Verifier,
+    FAILURE_MESSAGE_INDICATOR, MESSAGE_INDENT, SUCCESS_MESSAGE_INDICATOR,
+};
```

Comment:
Knowing that quote specific verifications are coming, I feel like these should get moved to another module in the crate.